devops/scripts/fetchOldTags.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# Usage: $0 --license_plate=VALUE --env=VALUE --app_name=VALUE [--prefix=VALUE] [--min_tags=VALUE]
+
+# Initialize default values
+LICENSE_PLATE=""
+ENV=""
+APP_NAME=""
+PREFIX=""
+MIN_TAGS=10  # Default to keeping all tags if MIN_TAGS is not specified
+
+# Parse named arguments
+while [ $# -gt 0 ]; do
+  case "$1" in
+    --license_plate=*)
+      LICENSE_PLATE="${1#*=}"
+      ;;
+    --env=*)
+      ENV="${1#*=}"
+      ;;
+    --app_name=*)
+      APP_NAME="${1#*=}"
+      ;;
+    --prefix=*)
+      PREFIX="${1#*=}"
+      ;;
+    --min_tags=*)
+      MIN_TAGS="${1#*=}"
+      ;;
+    *)
+      echo "Invalid argument: $1"
+      echo "Usage: $0 --license_plate=VALUE --env=VALUE --app_name=VALUE [--prefix=VALUE] [--min_tags=VALUE]"
+      exit 1
+  esac
+  shift
+done
+
+# Validation of required arguments and env values
+if [ -z "$LICENSE_PLATE" ] || [ -z "$ENV" ] || [ -z "$APP_NAME" ]; then
+  echo "License plate, environment, and app name are required."
+  echo "Usage: $0 --license_plate=VALUE --env=VALUE --app_name=VALUE [--prefix=VALUE] [--min_tags=VALUE]"
+  exit 1
+fi
+
+# Check if env value is allowed
+case "$ENV" in
+  dev|test|prod)
+    # All good, continue
+    ;;
+  *)
+    echo "The --env parameter must be one of: dev, test, prod."
+    exit 1
+    ;;
+esac
+
+# Setup internal variables
+DC_NAMESPACE="${LICENSE_PLATE}-${ENV}"
```

Comment:
Getting the namespace this way would be tricky because, our test and staging environment uses same name space.